### ISSUE TITLE
error when generating pdb

### ISSUE THREAD

**Issue by xxiexuezhi** [jonathanking/sidechainnet#48]
Dear talents,

I try to run the note book tutorials regarding using angles to form pdb files. But I keep getting errors like " Can't call numpy() on Tensor that requires grad. Use tensor.detach().numpy() instead." the screen shot is show as below. thx a lot for your help.

![Screenshot from 2022-07-31 13-11-42](https://user-images.githubusercontent.com/23486817/182037815-d4ce6bd5-6dd7-429d-bdb1-c5b7c1c3d872.png)
![image](https://user-images.githubusercontent.com/23486817/182037878-43640566-f155-4ea3-980f-68849ea1561b.png)

**Comment by jonathanking** [comment]
Thanks for bringing this to my attention! I'll take a look at get back to you ASAP.

**Comment by jonathanking** [comment]
Hi, sorry for the long delay. That should fix the issue. Let me know if I can do anything else!
